module.exports = {
  1: "There was a generic server error.",
  2: "Internal logic error.",
  3: "Access permission to the server was denied.",
  4: "Callback routine requested an abort.",
  5: "The database file is locked and cannot be accessed.",
  6: "A table in the database is locked and cannot be accessed.",
  7: "A malloc() has failed.",
  8: "Attempt to write to a read-only database has failed.",
  9: "The operation was terminated by sqlite3_interrupt.",
  10: "A disk I/O error occurred while accessing the database.",
  11: "The database disk image is malformed.",
  12: "Unknown opcode in sqlite3_file_control().",
  13: "The insertion failed because the database is full.",
  14: "Unable to open the database file.",
  15: "The database lock threw a protocol error.",
  16: "Internal use only.",
  17: "The string or BLOB exceeds the size limit available to it.",
  18: "The database scheme has changed.",
  19: "The request aborted due to a constraint violation.",
  20: "There was a data type mismatch.",
  21: "The library was used incorrectly.",
  22: "OS features were used that are not supported on the host.",
  23: "Authorization for accessing the database was denied.",
  24: "Not used.",
  25: "The 2nd parameter to sqlite3_bind was out of range.",
  26: "A file was opened that is not a database file.",
  27: "Notifications from sqlite3_log().",
  28: "Warning from sqlite3_log().",
  100: "sqlite3_step() has another row ready.",
  101: "sqlite3_step() has finished executing."
};